styled.stylelintrc.js

```diff
@@ -75,6 +75,6 @@ module.exports = {
     "no-empty-source": null,
     indentation: null,
     "max-empty-lines": 3,
-    "no-eol-whitespace": true,
+    "no-eol-whitespace": null,
```

Comment:
was throwing too many false positives and seemed to be clashing hard with prettier config, so disabling this rule for now.